README.md

```diff
@@ -682,6 +683,19 @@ Example:
 ./pkb.py --cloud=GCP --zone=us-central1-a --benchmarks=iperf,ping
 ```
 
+The disk type names vary by provider, but the following table summarizes some
+useful ones.
+
+Cloud name | Network-attached SSD | Network-attached HDD
```

Comment:
There's a note below the table about local SSD. The `local` disk type is special.